### PR TITLE
Use LogVerbose to replace otInfo: LegacySymmetric.cpp

### PR DESCRIPTION
**Pull request by Wallydog66**
Passes all tests.

Signed-off-by: Wallydog66 <wallydog.66@gmail.com>

### CODE REVIEW

**Comment by Wallydog66** [comment]
These are all fixed

File: src/core/recurring/OTAgreement.cpp [hunk 1]
```diff
@@ -1201,18 +1201,16 @@ std::int32_t OTAgreement::ProcessXMLNode(irr::io::IrrXMLReader*& xml)
...
-               << strInstrumentDefinitionID << "\n NotaryID: " << strNotaryID
-               << "\n"
-                  " senderAcctID: "
-               << strSenderAcctID << "\n senderNymID: " << strSenderNymID
-               << "\n "
-                  " recipientAcctID: "
-               << strRecipientAcctID
-               << "\n recipientNymID: " << strRecipientNymID << "\n ";
+        LogVerbose(OT_METHOD)(__FUNCTION__)
+               (" Creation Date: ") (tCreation)
```
Comment (FellowTraveler) [inline src/core/recurring/OTAgreement.cpp]: ```(" Creation Date: ") (tCreation)```

SHOULD BE:

```(": Creation Date: ") (tCreation)```

File: src/core/script/OTScriptable.cpp [hunk 2]
```diff
@@ -2715,9 +2715,9 @@ std::int32_t OTScriptable::ProcessXMLNode(irr::io::IrrXMLReader*& xml)
                     }
 
                     if (AddParty(*pParty))
-                        otInfo << szFunc
-                               << ": Loaded Party: " << pParty->GetPartyName()
-                               << "\n";
+                        LogVerbose(OT_METHOD)(__FUNCTION__)(szFunc)
```
Comment (FellowTraveler) [inline src/core/script/OTScriptable.cpp]: Remove the ```(szFunc)```

***We are getting rid of szFunc*** and replacing it with ```(OT_METHOD)(__FUNCTION__)```

So in this case you added the right thing, but you failed to remove the old thing.

File: src/core/script/OTSmartContract.cpp [hunk 3]
```diff
@@ -5280,14 +5281,12 @@ std::int32_t OTSmartContract::ProcessXMLNode(irr::io::IrrXMLReader*& xml)
...
-        otInfo << " Creation Date: " << tCreation
-               << "   Valid From: " << tValidFrom << "\n Valid To: " << tValidTo
-               << "\n"
-                  " NotaryID: "
-               << strNotaryID
-               << "\n"
-                  " activatorNymID: "
-               << strActivatorNymID << "\n ";
+        LogVerbose(OT_METHOD)(__FUNCTION__)
+               (" Creation Date: ") (tCreation)
```
Comment (FellowTraveler) [inline src/core/script/OTSmartContract.cpp]: ```(" Creation Date: ") (tCreation)```

SHOULD BE:

```(": Creation Date: ") (tCreation)```

File: src/crypto/key/LegacySymmetric.cpp [hunk 4]
```diff
@@ -434,8 +434,8 @@ bool LegacySymmetric::ChangePassphrase(
     OT_ASSERT(m_bIsGenerated);
 
     // Todo: validate the passphrases exist or whatever?
-    otInfo << "  Begin: " << __FUNCTION__
-           << ": Changing password on symmetric key...\n";
+    LogVerbose(OT_METHOD)(__FUNCTION__) ("  Begin: ")
```
Comment (FellowTraveler) [inline src/crypto/key/LegacySymmetric.cpp]: ```LogVerbose(OT_METHOD)(__FUNCTION__) (" Begin: ")```

SHOULD BE:

```LogVerbose(OT_METHOD)(__FUNCTION__) (": Begin: ")```

File: src/crypto/key/LegacySymmetric.cpp [hunk 5]
```diff
@@ -509,9 +509,9 @@ bool LegacySymmetric::ChangePassphrase(
         encrypted_key_);  // OUTPUT. (Ciphertext.)
     m_bIsGenerated = bEncryptedKey;
 
-    otInfo << "  End: " << __FUNCTION__
-           << ": (Changing passphrase on symmetric key...) "
-           << (m_bIsGenerated ? "SUCCESS" : "FAILED") << "\n";
+    LogVerbose(OT_METHOD)(__FUNCTION__) ("  End: ")
```
Comment (FellowTraveler) [inline src/crypto/key/LegacySymmetric.cpp]: ```LogVerbose(OT_METHOD)(__FUNCTION__) (" End: ")```

SHOULD BE:

```LogVerbose(OT_METHOD)(__FUNCTION__) (": End: ")```

File: src/crypto/key/LegacySymmetric.cpp [hunk 6]
```diff
@@ -536,8 +536,8 @@ bool LegacySymmetric::GenerateKey(
     OT_ASSERT(!m_bIsGenerated);
 
     Lock lock(lock_);
-    otInfo << "  Begin: " << __FUNCTION__
-           << ": GENERATING keys and passwords...\n";
+    LogVerbose(OT_METHOD)(__FUNCTION__) ("  Begin: ")
```
Comment (FellowTraveler) [inline src/crypto/key/LegacySymmetric.cpp]: Missing colon start of string.

File: src/crypto/key/LegacySymmetric.cpp [hunk 7]
```diff
@@ -610,9 +610,9 @@ bool LegacySymmetric::GenerateKey(
         encrypted_key_);  // OUTPUT. (Ciphertext.)
     m_bIsGenerated = bEncryptedKey;
 
-    otInfo << "  End: " << __FUNCTION__
-           << ": (GENERATING keys and passwords...) "
-           << (m_bIsGenerated ? "SUCCESS" : "FAILED") << "\n";
+    LogVerbose(OT_METHOD)(__FUNCTION__) ("  End: ")
```
Comment (FellowTraveler) [inline src/crypto/key/LegacySymmetric.cpp]: Missing colon.

**Pull request merged**